src/types/common.tsx

```diff
@@ -0,0 +1,50 @@
+export type QueryParams = Record<string, string>;
+
+export type Breed = {
+  weight: {
```

Comment:
maybe a good idea to create a specific types for nested objects values.
```
type Weight = {
  impertial: string,
  metric: string,
}
```

Reply:
done